src/libs/actions/FormActions.js

```diff
@@ -24,16 +24,7 @@ function setDraftValues(formID, draftValues) {
     Onyx.merge(`${formID}DraftValues`, draftValues);
```

Comment:
After testing again, we're having the same [issue](https://github.com/Expensify/App/issues/11510) where we're not sending the `accountNumber` and `routingNumber`:

https://user-images.githubusercontent.com/6829422/195201547-3dea3683-f4a1-42a1-955e-7058f22394e9.mov

I think we should only use `reimbursementAccountDraft`to store those values

```suggestion
    Onyx.merge(`${formID}Draft`, draftValues);
```